pos_mobile/models/pos_mobile_models.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from odoo import models, api
+
+
+class PosConfig(models.Model):
+    _inherit = 'pos.config'
```

Comment:
It's better to name this file as pos_config.py
see https://www.odoo.com/documentation/10.0/reference/guidelines.html